terraform/modules/tf_threat_intel_downloader/dynamodb.tf

```diff
@@ -18,3 +18,80 @@ resource "aws_dynamodb_table" "threat_intel_ioc" {
     Name = "StreamAlert"
   }
 }
+
+// IAM Role: Application autoscalling role
+resource "aws_iam_role" "appautoscaling" {
```

Comment:
`appautoscaling` is not super descriptive within the context of StreamAlert

Reply:
I rename it to `stream_alert_dynamodb_appautoscaling`